ui/app/AppLayouts/Profile/panels/ContactPanel.qml

```diff
@@ -51,6 +51,28 @@ StatusListItem {
     signal rejectionRemoved(string publicKey)
     signal textClicked(string publicKey)
 
+    subTitle: Utils.getElidedCompressedPk(container.publicKey)
+
+    image {
+        width: 40
+        height: 40
+        source: container.iconSource
+    }
+
+    icon {
+        width: 40
+        height: 40
+        color: Utils.colorForPubkey(container.publicKey)
+        letterSize: Math.max(4, root.image.width / 2.4)
+        charactersLen: 2
+        isLetterIdenticon: !container.image.source.toString()
+    }
+
+    ringSettings {
+        ringSpecModel: Utils.getColorHashAsJson(container.publicKey)
+        ringPxSize: Math.max(icon.width / 24.0)
+    }
```

Comment:
@borismelnik I know there's some work by @alexandraB99 tha tmerges `imageSettings` and `iconSettings`. I don't think those have landed yet, but just mentioning this here so we don't miss updating this part again.